src/enkf/gridinfo_nmmb.f90

```diff
@@ -124,6 +126,8 @@ subroutine getgridinfo(fileprefix, reducedgrid)
    allocate(latsgrd(npts),lonsgrd(npts))
    allocate(logp(npts,nlevs_pres)) ! log(ens mean first guess press) on mid-layers
    allocate(gridloc(3,npts))
+   allocate(taper_vert(nlevs))
```

Comment:
Since taperanalperts =.true. won't have actual impacts for gridinfo_wrf.f90 and gridinfo_nmmb.f90 (taper_vert always=1) . Shall we add a test in those program, when taperanalperts=.true.,  the program would abort with error message to avoid any mis-use of taperanalperts=.true. (while the users think they use while not) for those regional applications.

Reply:
It appears that the wrf and nmmb versions of the code have bitrot - they don't compile (due to missing parallel read/write functions).  Not sure if they are maintained anymore.